app/controllers/admin/youtube_playlists_controller.rb

```diff
@@ -2,4 +2,27 @@ class Admin::YoutubePlaylistsController < Admin::BaseController
   def new
     @admin = current_user
   end
+
+  def create
+    if results.snippet.nil?
+      flash[:error] = 'Sorry, that ID is not valid. Try again?'
+      redirect_to new_admin_youtube_playlist_path
+    else
+      tutorial_from_playlist = Tutorial.create(new_tutorial_params)
+      tutorial_from_playlist.create_playlist_videos
+      context = view_context.link_to('View it here', tutorial_path(tutorial_from_playlist.id))
+      flash[:success] = "Successfully created tutorial. #{context}."
+      redirect_to admin_dashboard_path
+    end
```

Comment:
your code that was moved to this controller to make room for manual tutorial

Reply:
ding ding ding. Thanks this is such a smart solve.